app/common/src/utilities/data/object.ts

```diff
@@ -9,6 +9,28 @@ export type Mutable<T> = {
   -readonly [K in keyof T]: T[K]
 }
 
+/** Whether two values are deeply equal. */
```

Comment:
I think the docs should be extended, as "deeply equal" is unprecise in js world.

This very instance, for example, will consider two objects of different classes equal if they happen to have same properties.

Also, there's already an existing method in [lodash library](https://lodash.com/docs/4.17.15#isEqual) which AFAIK we have an indirect dependency to anyway.

Reply:
added [fast-deep-equal](https://www.npmjs.com/package/fast-deep-equal) instead. note that it's no longer actually used currently but i don't think it hurts to add this anyway for posterity. thoughts?

Reply:
I don't see much point, unless you actually see that it will be needed soon.

Reply:
yeah fair enough. i don't actually see it being needed